tests/test.py

```diff
@@ -0,0 +1,4 @@
+bla1
```

Comment:
W0104 (pylint): Statement seems to have no effect (pointless-statement)  <!-- Automatically posted by Lintly -->